Azure/azure-application-gateway/ingest/parser.yml

```
@@ -0,0 +1,51 @@
+name: azure-application-gateway
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+
+  - name: parsed_date
+    external:
+      name: date.parse
+      properties:
+        # Maybe also it should be just `time` field in original message
+        input_field: "{{json_event.message.timeStamp}}"
+        output_field: date
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: ["network"]
+          event.type: ["access", "connection"]
+          "@timestamp": "{{parsed_date.date}}"
+          event.provider: "Azure"
+          event.action: "{{json_event.message.operationName}}"
+          event.dataset: "Azure Application Gateway"
+          event.reason: "{{json_event.message.properties.message}}"
+          source.ip: "{{json_event.message.properties.clientIP or json_event.message.properties.clientIp}}"
+          source.port: "{{json_event.message.properties.clientPort}}"
+          source.bytes: "{{json_event.message.properties.receivedBytes}}"
+          destination.bytes: "{{json_event.message.properties.sentBytes}}"
+          network.bytes: "{{json_event.message.properties.get('receivedBytes', 0) + json_event.message.properties.get('sentBytes', 0)}}"
+          user_agent.original: "{{json_event.message.properties.userAgent}}"
+          http.request.method: "{{json_event.message.properties.httpMethod}}"
+          http.response.status_code: "{{json_event.message.properties.httpStatus}}"
+          url.original: "{{json_event.message.properties.originalRequestUriWithArgs}}"
+
+          azure.application_gateway.transactionId: "{{json_event.message.properties.transactionId}}"
+          azure.application_gateway.error_info: "{{json_event.message.properties.error_info}}"
+          azure.application_gateway.serverStatus: "{{json_event.message.properties.serverStatus}}"
+          azure.application_gateway.sslClientVerify: "{{json_event.message.properties.sslClientVerify}}"
+          azure.application_gateway.message: "{{json_event.message.properties.message}}"
+          azure.application_gateway.details.message: "{{json_event.message.properties.details.message}}"
+          azure.application_gateway.instanceId: "{{json_event.message.properties.instanceId}}"
+          azure.application_gateway.action: "{{json_event.message.properties.action}}"
```

`event.action` seems more suitable for this field
```suggestion
          event.action: "{{json_event.message.properties.action}}"
```